hack/chains/cosign-verify-tasks.yaml

```diff
@@ -0,0 +1,88 @@
+---
```

Comment:
Should we move all tasks to the build-definitions repo? https://github.com/redhat-appstudio/build-definitions/tree/main/tasks

Reply:
Yes! Especially now that we have redhat-appstudio/build-definitions#28 merged there, we should bring this over there. I wanted a way for folk to try this out and provide feedback, I wasn't aware of `hack/build/utils/install-pipelines.sh` as a way of testing this out.